README.md

```diff
@@ -73,17 +73,6 @@ subgraph ekstern["NAV felles"]
 end
 ```
 
-# Kom i gang
-
-Noen avhengigheter i prosjektet ligger i Github Package Registry som krever autentisering. Det enkleste er å lage
-en [PAT (Personal Access Token)](https://github.com/settings/tokens).
-
-1. [Opprett PAT her](https://github.com/settings/tokens). I tilfelle lenken ikke fungerer går man
-   til `Github -> Settings -> Developer settings -> Personal access tokens`
-2. Huk av `read:packages`. Ikke legg til flere scopes enn nødvendig.
-3. Autoriser navikt-organisasjonen for SSO ved å velge "Configure SSO" på tokenet
-4. Tokenet legges i `.zshrc` med `export GITHUB_TOKEN=<token>`
-
```

Comment:
Pakken for dekoratøren ligger på NavIKT sin github package repository, så tror ikke denne kan fjernes enda

Reply:
Åja, såg bare på gradle bruken.
Men da burde man jo vært slik:
`export NPM_TOKEN=<token>` ?

Reply:
NPM pakken dems referer videre til GitHub Package Registry, så tror det er derfor vi fortsatt trenger GitHub token for å autentisere oss mot navikt sin github org

https://www.npmjs.com/package/@navikt/nav-dekoratoren-moduler

Reply:
Riktig, det mener eg også.
Men miljøvariablen er vell NPM_TOKEN? og ikkje GITHUB_TOKEN
ref: 
https://github.com/navikt/pensjon-etterlatte/blob/77384c782a129e4a9828f2c4809c9e202192e864/apps/etterlatte-node-server/.npmrc#L3

Reply:
åja se der ja. da henger jeg med. Ja NPM_TOKEN trenger vi, men da trenger vi ikke GITHUB_TOKEN for dekoratøren ivertfall

Reply:
Men om vi trenger det får gradle får noen backend-boys svare på @Watercolours @oyvindsh @sebassonav